cloudy-native/src/main/cpp/JniEntryPoints.cpp

```
@@ -219,6 +225,10 @@ Java_com_skydoves_cloudy_internals_render_RenderScriptToolkit_nativeBlur(
     RestrictionParameter restrict{env, restriction};
     ByteArrayGuard input{env, input_array};
     ByteArrayGuard output{env, output_array};
+    if (!input.valid() || !output.valid()) {
+        ALOGE("nativeBlur: failed to access input/output byte array");
+        return;
+    }
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)